### DISCUSSION TITLE
Why Isn't see autocomplete?

### DISCUSSION THREAD

**Discussion by bilyalov** [collerek/ormar#244] [Q&A]
![image](https://user-images.githubusercontent.com/25269937/121949610-7a825c80-cd7a-11eb-9689-b72aa91dc7ec.png)

Help me pls

**Comment by collerek** [comment]
There is some problems with pycharm autocompletion.

Works as expected in vscode:
![image](https://user-images.githubusercontent.com/16324238/122237096-57ad9080-cebf-11eb-9e5c-a8b41729d1e6.png)

I created a bug report for JetBrains team:
https://youtrack.jetbrains.com/issue/PY-49306
